### DISCUSSION TITLE
New endpoint Assessment service /question/v1/bulk/upload {"message":"no route and no API found with those values"}

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#392] [Installation] [answered]
CSP:GCP
Release-5.1.0

Recently we have added new endpoint in the  Kong API for the Assessment service /question/v1/bulk/upload

- name: questionBulkUpload
  uris: "{{ question_prefix }}/v1/bulk/upload"
  upstream_url: "{{ assessment_service_url }}/question/v4/bulk/upload"
  strip_uri: true
  plugins:
  - name: jwt
  - name: cors
  - "{{ statsd_pulgin }}"
  - name: acl
    config.whitelist:
    - contentCreate
  - name: rate-limiting
    config.policy: local
    config.hour: "{{ medium_rate_limit_per_hour }}"
    config.limit_by: credential
  - name: request-size-limiting
    config.allowed_payload_size: "{{ medium_request_size_limit }}"
  - name: opa-checks
    config.required: false
    config.enabled: false

Added in the following path ansible/roles/kong-api/defaults/main.yml

Deployed the below service
- APIManager
- OnboardAPI

still facing issue with {"message":"no route and no API found with those values"}

Troubleshoot done so far

- Check inside APImanager endpoints is available
- using nginx-privateIP it works
- inside assessment service, it works

through domain, it throw {"message":"no route and no API found with those values"

**Comment by Krishnaj20** [comment]
@pallakartheekreddy - Please check this query and respond.

@surendrasinghs @vinukumar-vs

**Comment by gandham-santhosh** [comment, marked as answer]
@tsprasath Did you check if the API is onboarded to the kong? you can run http://localhost:8001/apis to check if this API is onboarded. How are you referring the API?

**Reply by tsprasath** [reply]
@gandham-santhosh  this issue got resolved. Endpoint URL was wrongly  hitted from the domain
